static/docs/command-reference/run.md

```diff
@@ -72,6 +72,14 @@ must produce the same results given the same inputs/dependencies).
 Have in mind what brings entropy to your command (e.g. random generators, time,
 hardware, etc.) and try to minize it (e.g. fix seeds).
 
+### Locking
+
+When running your command, DVC will release its repository lock (.dvc/lock), so
```

Comment:
make .dvc/lock -> `.dvc/lock`

btw, do we still use a single file to lock? (haven't had a chance still to take a look at the PR)

Reply:
repo lock is one and only. For rwlocks we use .dvc/tmp/rwlock

Reply:
```suggestion
After executing the `command`, DVC will release its repository lock (`.dvc/lock`), so
```

But we should probably mention that `.dvc/lock` is created while the command is running somewhere above in the main description.

Reply:
In fact maybe you don't need a separate section for this one paragraph, just add a paragraph in the description, before **Well-behaved commands**.

Reply:
Or maybe I just don't understand how locking works now? (After reading the rest of the paragraph.) Is there an explanation somewhere? 🙂

Reply:
@jorgeorpinel I've commited your suggestion accidentally. I don't agree with it, `after executing` is not the same as `when running`. Also releasing is not the same as removing.

Reply:
OK, but I still see my text in there, do you want to update it? Let me re-open this discussion on the new code...

Reply:
See https://github.com/iterative/dvc.org/pull/868#pullrequestreview-335680058, thanks.